api/client/proto/authservice.proto

```diff
@@ -358,6 +358,62 @@ message CreateResetPasswordTokenRequest {
     int64 TTL = 3 [ (gogoproto.jsontag) = "ttl", (gogoproto.casttype) = "Duration" ];
 }
 
+// CreateBotJoinTokenRequest is a request to create a reset password token.
+message CreateBotJoinTokenRequest {
+    // Name is the bot's user name.
+    string Name = 1 [ (gogoproto.jsontag) = "name" ];
+    // TTL specifies how long the generated token is valid for.
+    int64 TTL = 2 [ (gogoproto.jsontag) = "ttl", (gogoproto.casttype) = "Duration" ];
+}
+
+// RenewableCertsRequest is a request to generate a first set of renewable
+// certificates from a bot join token.
+message RenewableCertsRequest {
+    // Token is a bot join token.
+    string Token = 1 [ (gogoproto.jsontag) = "token" ];
```

Comment:
Nope, for IAM joining you still pass along a token name, but rather than a random/secure value it's defined in advance.

Reply:
Should probably clarify this more - we'll be outright removing `RenewableCertsRequest` and `GenerateInitialRenewableUserCerts` with @nklaassen's work on IAM joining since it'll join using the usual `auth.Register()` flow.